### ISSUE TITLE
Best of OSM Belgium

### ISSUE THREAD

**Issue by nobohan** [osmbe/community_planning#37]
"Best of OSM Belgium" would be a project where we present some places that are very well mapped in details in OSM in Belgium, for a particular topic. The goal would be to present the OSM data to potential end-users: public administrations, companies, associations.  Sometimes, very well mapped places can be the result of a collective coordinate effort, an uncoordinate collective effort, or purely due to a strong mapping addiction of a particular individual. In the latter case, I see it as an extension of the mapper of the month #24. 

Below are some examples, feel free to add some! 

Examples

1. Buildings in Flanders (once the GRB import has been completed ;-) )
2. Sidewalks in Brugelette
3. Forest tracks in Forêt d'Anlier
4. Indoor map in ? (LLN, KUL, ?)
5. Gardens in Krainem
6. Mobile phone antennas in Belgium (complete since an import of Anakil, right? )
7. PT network in ? 
8. Knooppunten cycle network in ? 
9. shops in ??? (best commercial area / commercial city centre)
10. 3D buildings in ? 

Public:

- public administrations: to have some information about how OSM data can be (safely) used
- private companies / NGO / association: to have some information about how OSM data can be (safely) used
- Others OSM mappers, to have some inspiration

How/format: It can be some pages on osm.be, with a webmap  or not. Each topic could be described with some text, but also some "fields" to fill if possible:

- who did it: collective or a list of usernames
- where: a bbox (can be used for a webmap then)
- which tags: a list of `key=value`
- overpass-query
- alternative/authoritative dataset
- contact (if you want to know more about the data quality)
- an image
- use: some links to some apps that use these kind of data, for instance indoor map viewer for the indoor example, cycling.waymarkedtrails for the Knooppunten example, 3D maps for the 3D building example

BTW there still exist https://bestofosm.org/, but this is old and not as much as detailed as what I propose here. 

That's it. Easy to open an issue. More difficult to close it!

**Comment by thibaultmol** [comment]
Not sure how useful, but I came across this github page which I kind of liked how they show embedded map example (all examples on one page)
http://jessecrocker.github.io/WhatIsOpenStreetMap/

**Comment by nobohan** [comment]
yes, that makes me think that each topic could be illustrated by a specific slippy map: a 3D map, a hiking map, etc...

**Comment by pietervdvn** [comment]
Does it have to be very serious? [If not, I have a proposal](https://pietervdvn.github.io/MapComplete/index.html?z=12&lat=51.205&lon=3.2636&background=osm&userlayout=true#eyJpZCI6ImlkIiwidGl0bGUiOnsibmwiOiJGcml0dXJlbmthYXJ0IiwiZnIiOiJDYXJ0ZSBkZXMgZnJpdHVyZXMifSwiZGVzY3JpcHRpb24iOnt9LCJsYW5ndWFnZSI6WyJubCIsImZyIl0sIm1haW50YWluZXIiOiIiLCJpY29uIjoiaHR0cHM6Ly91cGxvYWQud2lraW1lZGlhLm9yZy93aWtpcGVkaWEvY29tbW9ucy81LzU1L0ZyZW5jaF9mcmllc19qdWxpYW5lX2tyX3Iuc3ZnIiwidmVyc2lvbiI6IjAiLCJzdGFydExhdCI6MCwic3RhcnRMb24iOjAsInN0YXJ0Wm9vbSI6MSwid2lkZW5GYWN0b3IiOjAuMDUsInNvY2lhbEltYWdlIjoiIiwibGF5ZXJzIjpbeyJpZCI6ImZyaXR1cmVzIiwibmFtZSI6IkxheWVyIiwibWluem9vbSI6OCwib3ZlcnBhc3NUYWdzIjp7ImFuZCI6WyJjdWlzaW5lPWZyaXR1cmUiXX0sInRpdGxlIjp7InJlbmRlciI6eyJubCI6IkZyaXR1dXIiLCJmciI6IkZyaXR1cmUifSwibWFwcGluZ3MiOlt7ImlmIjp7ImFuZCI6WyJuYW1lfioiXX0sInRoZW4iOnsibmwiOiJGcml0dXVyIDxpPntuYW1lfTwvaT4iLCJmciI6IkZyaXR1cmUgPGk+e25hbWV9PC9pPiJ9fV19LCJkZXNjcmlwdGlvbiI6e30sInRhZ1JlbmRlcmluZ3MiOltdLCJpY29uIjp7InJlbmRlciI6Imh0dHBzOi8vdXBsb2FkLndpa2ltZWRpYS5vcmcvd2lraXBlZGlhL2NvbW1vbnMvNS81NS9GcmVuY2hfZnJpZXNfanVsaWFuZV9rcl9yLnN2ZyJ9LCJ3aWR0aCI6eyJyZW5kZXIiOiI4In0sImljb25TaXplIjp7InJlbmRlciI6IjQwLDQwLGNlbnRlciJ9LCJjb2xvciI6eyJyZW5kZXIiOiIjMDBmIn0sInByZXNldHMiOltdLCJ3YXlIYW5kbGluZyI6MX1dLCJyb2FtaW5nUmVuZGVyaW5ncyI6W119)
